### PR TITLE
add glamorous-tailwind example

### PR DESCRIPTION
**Pull request by deden**
Add Tailwind CSS example with CSS-in-JS by using glamor-glamorous.

### CODE REVIEW

File: examples/glamorous-tailwind/src/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// Render your app
+if (typeof document !== 'undefined') {
+  const render = Comp => {
+    ReactDOM.hydrate(
+      <AppContainer>
+        <Comp />
+      </AppContainer>,
+      document.getElementById('root'),
+    )
+  }
```
Comment (EmilTholin) [inline examples/glamorous-tailwind/src/index.js]: Awesome stuff!
Could you change this to [reflect the other examples](https://github.com/nozzle/react-static/blob/master/examples/basic/src/index.js#L13-L21)?

**Comment by EmilTholin** [comment]
Really cool stuff.
I have no Tailwind experience, but do you think the [errors in the build](https://travis-ci.org/nozzle/react-static/builds/311809170?utm_source=github_status&utm_medium=notification) can be fixed easily?

File: examples/glamorous-tailwind/src/index.js (hunk 1 above)
Comment (deden) [reply examples/glamorous-tailwind/src/index.js]: ok, done. Sorry, I missed that update.

**Comment by deden** [comment]
tailwind.js config file was created by Tailwind-cli utility. Can we just disable it with `/* eslint-disable */` ?

**Comment by EmilTholin** [comment]
Yeah, I don't think that will be a problem. You could alternatively run `yarn test --fix` and see if `eslint` can change it automatically.

**Comment by viperfx** [comment]
@deden I have been trying to follow this example template but use it with styled components. Could you give a bit of info about how glamour is injecting tailwind and how the css is actually served up? I could not figure out by reading the code.

**Pull request merged**